### DISCUSSION TITLE
inav no configuration received within 10 seconds, communication failed #6574

### DISCUSSION THREAD

**Discussion by sbrown123** [iNavFlight/inav#7148] [Q&A]
Cannot get a connection to f722 hd board. getting error message above. Using Mac. 

![Screen Shot 2021-05-25 at 2 59 33 PM](https://user-images.githubusercontent.com/5177590/119553672-d8b6b380-bd69-11eb-8792-ac3067c05463.png)

**Comment by DzikuVx** [comment]
I see you use Bluetooth, is this thing configured correctly?
